src/components/Select/utils.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 
 import {KeyCode} from '../../constants';
-import type {List, ListItemData} from '../List';
+import {List, type ListItemData} from '../List';
```

Comment:
Let's use types import separately. This import corrupts code in case of using typescript below version 4.5